projects/l2_pooling/capacity_test.py

```diff
@@ -165,25 +166,33 @@ def testNetworkWithOneObject(objects, exp, testObject, numTestPoints):
   :param numTestPoints: number of test points on the test object
   :return:
   """
-  numObjects = len(objects.getObjects())
-  numPointsPerObject = len(objects.getObjects()[0])
+  innerObjs = objects.getObjects()
+  numObjects = len(innerObjs)
+  numPointsPerObject = len(innerObjs[0])
 
   testPts = np.random.choice(np.arange(numPointsPerObject),
                              (numTestPoints,),
                              replace=False)
+
   testPairs = [objects[testObject][i] for i in testPts]
 
   exp._unsetLearningMode()
   exp.sendReset()
 
-  overlap = np.zeros((numTestPoints, numObjects))
+  # Split test points between columns
+  numTestPointsPerColumn = numTestPoints / exp.numColumns
+
+  overlap = np.zeros((numTestPointsPerColumn, numObjects))
+
+  for step, pairs in enumerate(zip(*zip(*[iter(testPairs)] * (len(testPairs) / exp.numColumns)))):
+    for colIdx in xrange(exp.numColumns):
```

Comment:
I've since added a comment above this to explain how it's iterating, but essentially yes.  If there are more than one column, each gets a non-overlapping sequence of feature-location pairs.